src/extractor/edge_based_graph_factory.cpp

```diff
@@ -138,7 +138,7 @@ void EdgeBasedGraphFactory::InsertEdgeBasedNode(const NodeID node_u, const NodeI
     NodeID current_edge_source_coordinate_id = node_u;
 
     // traverse arrays from start and end respectively
-    for (const auto i : util::irange(0UL, geometry_size))
+    for (const auto i : util::irange(std::size_t{ 0 }, geometry_size))
```

Comment:
Exactly, both types to `irange` have to be the same. On 64 bit Unix `size_t` is 64 bit / 8 bytes therefore the types match, but on 64 bit Windows the types are different.